docs/intro/tutorial.rst

```diff
@@ -323,33 +323,35 @@ use Firefox Developer Tools or some Firefox extensions like Firebug. For more
 information see :ref:`topics-firebug` and :ref:`topics-firefox`.
 
 After inspecting the page source, you'll find that the web site's information
-is inside a ``<ul>`` element, in fact the *second* ``<ul>`` element.
+is inside ``<div>`` elements. In fact, the title and description of the site's list
+are included in ``<div>`` elements that contain the ``class`` attribute
+``"title-and-desc"``.
 
-So we can select each ``<li>`` element belonging to the site's list with this
+So we can select each ``<div>`` element belonging to the site's list with this
 code::
 
-    response.xpath('//ul/li')
+    response.xpath('//div[@class="title-and-desc"]')
 
 And from them, the site's descriptions::
 
-    response.xpath('//ul/li/text()').extract()
+    response.xpath('//div[@class="title-and-desc"]/div/text()').extract()
```

Comment:
I would prefer that the tutorial shows using specific elements classes, like `<div class="site-descr ">` here,
e.g. `response.xpath('//div[@id="site-list-content"]/div[@class="site-item "]/div[@class="title-and-desc"]/div[@class="site-descr "]')`
(although it becomes quite ugly -- a reason to show how to mix/chain XPath and CSS)